packages/qgrid-ngx-plugins/src/lib/data-provider/data-provider-process-context.ts

```diff
@@ -0,0 +1,5 @@
+import { GridModel } from '@qgrid/ngx';
+
+export interface DataProviderProcessContext {
```

Comment:
can you rename it to DataProviderContext and put under the data-provider.ts file?

Reply:
done

Reply:
done